clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/AuditLoggingServerInterceptor.java

```diff
@@ -118,22 +119,7 @@ public <ReqT, RespT> Listener<ReqT> interceptCall(
       next.startCall(call, headers);
     }
 
-    log.info("Justification Required: " + auditLoggingConfiguration.isJustificationRequired());
-    if (auditLoggingConfiguration.isJustificationRequired()) {
-      log.info("Trying to add justification.");
-      try {
-        logBuilder = setJustification(headers, logBuilder);
-      } catch (LogProcessingException e) {
-        log.warn("Exception while trying to determine justification.");
-        if (ConfigUtils.shouldFailClose(auditLoggingConfiguration.getLogMode())) {
-          throw new IllegalStateException("Unable to determine justification.", e);
-        } else {
-          log.error(
-              "Justification was unable to be determined, continuing without audit logging.", e);
-          next.startCall(call, headers);
-        }
-      }
-    }
+    final Struct auditLogReqCtx = getAuditLogRequestContext(headers);
```

Comment:
In Java (in contrast to Go), we should avoid abbreviating names and spell them out as much as we can, so this one should be `final Struct auditLogRequestContext = ...`

Reply:
Ack, will change.